src/adapter/src/catalog/builtin.rs

```diff
@@ -1389,6 +1389,14 @@ pub static MZ_KAFKA_CONNECTIONS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable
         .with_column("sink_progress_topic", ScalarType::String.nullable(false)),
     is_retained_metrics_object: false,
 });
+pub static MZ_KAFKA_SOURCES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_kafka_sources",
+    schema: MZ_CATALOG_SCHEMA,
+    desc: RelationDesc::empty()
+        .with_column("id", ScalarType::String.nullable(false))
+        .with_column("consumer_group_id", ScalarType::String.nullable(false)),
```

Comment:
I think we should just call this `group_id`, as that's what Kafka's libraries call it, and that's also what we call it in the `GROUP ID PREFIX` option.

```suggestion
        .with_column("group_id", ScalarType::String.nullable(false)),
```